pkg/handlers/group.go

```diff
@@ -0,0 +1,110 @@
+package handlers
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"NKSS-backend/pkg/query"
+
+	"github.com/gorilla/mux"
+	"golang.org/x/exp/slices"
+)
+
+func GetAllMemberInfo(db *sql.DB, w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	w.Header().set("Content-Type", "application/json")
+
+	ctx := context.Background()
+	queries := query.New(db)
+	discord, err := queries.getAllGroups(ctx)
+	if err == sql.ErrNoRows {
+		respondError(w, 404, "No groups found!")
+		return
+	}
+	faculty , err := queries.getAllFaculty(ctx)
+	if err == sql.ErrNoRows {
+		respondError(w, 404, "No faculty found!")
+		return
+	}
+	social, err := queries.getAllGroupSocials(ctx)
+	if err == sql.ErrNoRows {
+		respondError(w, 404, "No socials found!")
+		return
+	}
+	admin, err := queries.getAllGroupAdmins(ctx)
+	if err == sql.ErrNoRows {
+		respondError(w, 404, "No admins found!")
+		return
+	}
+	member, err := queries.getAllGroupMembers(ctx)
+	if err == sql.ErrNoRows {
+		respondError(w, 404, "No members found!")
+		return
+	}
+	
+	var groups [](map[string]interface{})
+	
+	for g:=0; g < len(discord); g++ {
+		
+		var group map[string]interface{}
+
+		group["name"] := discord[g]["name"]
+		group["alias"] := discord[g]["alias"]
+		group["branch"] := discord[g]["branch"]
+		group["kind"] := discord[g]["kind"]
+		group["description"] := discord[g]["description"]
```

Comment:
The `group` variable is already declared, do not use `:=`; use `=` directly instead